src/platyPS/platyPS.psm1

```diff
@@ -2782,12 +2799,28 @@ function ConvertPsObjectsToMamlModel
     $Outputs = @()
 
     $Help.returnValues.returnValue | ForEach-Object {
-        $OutputObject = New-Object -TypeName Markdown.MAML.Model.MAML.MamlInputOutput
-        $OutputObject.TypeName = $_.type.name
-        $OutputObject.Description = $_.description |
-            DescriptionToPara |
-            AddLineBreaksForParagraphs
-        $Outputs += $OutputObject
+        if ($_.description -eq $null -and $_.type.name -ne $null)
+        {
+            $Outputtypes = $_.type.name.split("`n")
```

Comment:
You can use `It seems that `.Split("`n", [System.StringSplitOptions]::RemoveEmptyEntries)` to avoidempty check later.